src/builder.rs

```diff
@@ -71,13 +73,16 @@ pub struct ThingBuilder<Other: ExtendableThing, Status> {
     security_definitions: Vec<(String, UncheckedSecurityScheme)>,
     profile: Vec<String>,
     schema_definitions: HashMap<String, UncheckedDataSchemaFromOther<Other>>,
+
+    /// Thing extension.
```

Comment:
We might link to the extend module and put there a full explainer on another PR.